hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/VolumeArgs.java

```diff
@@ -18,13 +18,15 @@
 package org.apache.hadoop.ozone.om.helpers;
 
 import com.google.common.base.Preconditions;
+import net.jcip.annotations.Immutable;
 
 import java.util.HashMap;
 import java.util.Map;
 
 /**
  * A class that encapsulates the createVolume Args.
  */
+@Immutable
 public final class VolumeArgs {
```

Comment:
This `VolumeArgs` class is not used anywhere.  Let's remove it instead?

Reply:
Looks like I had changed the wrong class.  Thanks for spotting it.

Removed this one, annotated the real one.